src/main/app/src/pages/ScheduleActivityPage.js

```diff
@@ -162,14 +162,14 @@ const ScheduleActivityPage = props => {
       {displayErrors && (titleError || dateError) ? 
         <Alert severity="error" onClose={() => updateDisplayErrors(false)} className="mb-5">
           <AlertTitle>Error</AlertTitle>
-          {errors.map(errorObj =>
-              errorObj.error ? <div>{errorObj.errorMsg} — <strong>check it out!</strong></div> : null
+          {errors.map((errorObj, key) =>
+              errorObj.error ? <div key={key}>{errorObj.errorMsg} — <strong>check it out!</strong></div> : null
```

Comment:
Do you need the key twice? Did you want to pass in the key due to a react warning that it should be unique, or because you are using it anywhere?

Reply:
Yeah, it's to fix the warnings you get for not setting the key property when mapping a list.